src/azure-cli-core/azure/cli/core/auth/landing_pages/success.html

```diff
@@ -2,11 +2,32 @@
 <html>
 <head>
     <meta charset="utf-8" />
-    <meta http-equiv="refresh" content="10;url=https://docs.microsoft.com/cli/azure/">
+    <meta http-equiv="refresh" content="120;url=https://docs.microsoft.com/cli/azure/">
```

Comment:
```suggestion
    <meta http-equiv="refresh" content="60;url=https://docs.microsoft.com/cli/azure/">
```
2 minutes seems a bit long imo